locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -151,18 +153,18 @@ event loop and all of the asynchronous behaviors of the platform)
 also has a hard maximum (system dependent) before it stops polling for
 more events.
 
-### I/O callbacks
+### pending callbacks
 
 This phase executes callbacks for some system operations such as types
 of TCP errors. For example if a TCP socket receives `ECONNREFUSED` when
 attempting to connect, some \*nix systems want to wait to report the
-error. This will be queued to execute in the **I/O callbacks** phase.
+error. This will be queued to execute in the **pending callbacks** phase.
 
 ### poll
 
 The **poll** phase has two main functions:
 
-1. Executing scripts for timers whose threshold has elapsed, then
+1. Calculating how long it should block and poll for I/O, then
```

Comment:
`Calculating` doesn't cover the meaning of `Executing` here, and the main purpose of `poll` phase is to execute callbacks. So it'd be better to remain the same here or just modify `Executing scripts` to be `Executing callbacks` :)

Reply:
Yep, Node.js eventloop might differ a little from libuv's I/O loop, we set a [UV_RUN_ONCE](https://github.com/nodejs/node/blob/aec019030d9a2dc0d108eb22995bc379a7d0da08/deps/uv/src/unix/core.c#L372) mode to make sure at least one callback must be invoked when it returns. Maybe @tniessen could give a more detailed explanation, thx :)